sem/manager.py

```diff
@@ -318,6 +348,31 @@ def run_simulations(self, param_list, show_progress=True, stop_on_errors=True):
         if param_list == []:
             return
 
+            # check
```

Comment:
TODO?

Reply:
The comment was left out from the `parse_log_compoenent()` function. Please ignore this.